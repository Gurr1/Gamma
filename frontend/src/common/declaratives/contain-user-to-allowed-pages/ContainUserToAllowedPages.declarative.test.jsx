import React from "react";
import { shallow } from "enzyme";
import ContainUserToAllowedPages from "./ContainUserToAllowedPages.declarative";

describe("<ContainUserToAllowedPages/>", () => {
  test("Shallow render of <ContainUserToAllowedPages/>", () => {
    const wrapper = shallow(<ContainUserToAllowedPages />);
    expect(wrapper).toMatchSnapshot();
  });
});
