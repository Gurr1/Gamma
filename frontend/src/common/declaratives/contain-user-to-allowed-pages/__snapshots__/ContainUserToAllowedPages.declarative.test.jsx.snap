// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContainUserToAllowedPages/> Shallow render of <ContainUserToAllowedPages/> 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ContainUserToAllowedPages />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): null,
  Symbol(enzyme.__nodes__): Array [
    null,
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
